packages/loader/container-definitions/src/browserPackage.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidPackageEnvironment, IFluidPackage, isFluidPackage } from "./fluidPackage";
+
+/**
+ * A specific Fluid package environment for browsers
+ */
+export interface IFluidBrowserPackageEnvironment extends IFluidPackageEnvironment{
+    /**
+     * The umd target specifics the scripts necessary for loading a packages
+     * in a browser environment
+     */
+    umd: {
+        /**
+         * The bundled js files for loading this package
```

Comment:
If there is more than one file here, is the intention that these are in dependency (topological) order? We should probably call that out in the interface here so we can make sure container authors and code loaders respect this.

Reply:
we don't currently enforce any ordering here. we just ensure that window['library'] exists after loading all files.

Reply:
We may want to though. For example, we may want to make sure that code loaders evaluate the scripts in the order they are listed here. One example would be say we have two files : [fluidFramework.js, officeContainer.js] -> we'd want our code loader to load them in parallel, but evaluate the scripts serially such that fluidFramework.js is evlauated before office.container.js.

I believe FF already does preserve this ordering, since it adds the script tags with async false

Reply:
good point. I'll add some comments